database/insertData.js

```diff
@@ -442,6 +453,57 @@ const queries = [
                           "Producer"
                         ]
                       }
+                    },
+                    {
+                      code: "S1PB1"
+                      index: 3
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                      parameters: { previousValidityCheck: true }
+                    },
+                    {
+                      code: "S1T2"
+                      index: 4
+                      title: "Intro Section 1 - Page 2/2"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "Company nationality" }
+                    },
+                    {
+                      code: "S1Q3"
+                      index: 5
+                      title: "Organisation national or international"
+                      elementTypePluginCode: "radioChoice"
```

Comment:
For now can we use `dropdownChoice` so we can actually see the plugin on the page?

Reply:
Sure. I'm changing to this now.

Should be able to look have another look on [PR issue#126](https://github.com/openmsupply/application-manager-web-app/pull/132) today or tomorrow so we can see all running together. That will be nice indeed.